packages/ListBox/src/components/WithTags/WithTags.js

```diff
@@ -31,20 +32,31 @@ const propTypes = {
   renderPill: PropTypes.func,
   /** An array of id that helps the ListBoxWithTags to known what elements are selected  */
   selectedOptions: PropTypes.arrayOf(PropTypes.shape({})),
+  /** Provides an alternative for rendering the Pill label instead of using the default [{label:value}] coming from the og data */
+  pillLabelKey: PropTypes.string,
+  /** When this is true, it will display a message indicating all options are selected without showing the popover nor the search input */
+  allOptionsAreSelected: PropTypes.bool,
```

Comment:
Instead of passing in another prop, could you loop over the children and count how many are displayType `ListBox.Option`, and see if that count equals `selectedOptions.length`?

Reply:
No, I can't, I thought about it for some time, and couldn't figure out a way for knowing it, the children that are pass down are only the ones that are rendering either because a filter has been applying on top of them, or the initial list. I don't have control to know how big the list is or know what has been selected to be able to compared it. 

look at the story so you can see it.